### PR TITLE
Convert Numbers/Digits to their respective English representation without using library.

### PR DESCRIPTION
**Pull request by abhip99**
I have wrote the script to do this..

### CODE REVIEW

File: Basic-Scripts/int_to_en.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Aug 16 16:21:31 2020
+
+@author: Abhay Paliwal
+"""
+
+"""Given an int number, print it in English."""
+def int_to_en(num):
+    d = { 0 : 'zero', 1 : 'one', 2 : 'two', 3 : 'three', 4 : 'four', 5 : 'five',
```
Comment (SKAUL05) [inline Basic-Scripts/int_to_en.py]: Include Docstrings in your function definition.

File: Basic-Scripts/int_to_en.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+def int_to_en(num):
+    d = { 0 : 'zero', 1 : 'one', 2 : 'two', 3 : 'three', 4 : 'four', 5 : 'five',
+          6 : 'six', 7 : 'seven', 8 : 'eight', 9 : 'nine', 10 : 'ten',
+          11 : 'eleven', 12 : 'twelve', 13 : 'thirteen', 14 : 'fourteen',
+          15 : 'fifteen', 16 : 'sixteen', 17 : 'seventeen', 18 : 'eighteen',
+          19 : 'nineteen', 20 : 'twenty',
+          30 : 'thirty', 40 : 'forty', 50 : 'fifty', 60 : 'sixty',
+          70 : 'seventy', 80 : 'eighty', 90 : 'ninety' }
+    k = 1000
+    m = k * 1000
```
Comment (SKAUL05) [inline Basic-Scripts/int_to_en.py]: Use meaningful variable names.

File: Basic-Scripts/int_to_en.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        if (num % b) == 0: return int_to_en(num // b) + ' billion'
+        else: return int_to_en(num // b) + ' billion, ' + int_to_en(num % b)
+
+    if (num % t == 0): return int_to_en(num // t) + ' trillion'
+    else: return int_to_en(num // t) + ' trillion, ' + int_to_en(num % t)
+
+    raise AssertionError('num is too large: %s' % str(num))
+    
+
+number = int(input())
```
Comment (SKAUL05) [inline Basic-Scripts/int_to_en.py]: Include the input part in `main` function. Ask User for Input.

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
Follow [PEP8 Guidelines](https://www.python.org/dev/peps/pep-0008/) while coding.

**Comment by abhip99** [comment]
I have resolved the issue mention by @SKAUL05  , Can you review again my request to merge the code into main branch.

File: Basic-Scripts/int_to_en.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        else: return int_to_en(num // billion) + ' billion, ' + int_to_en(num % billion)
+
+    if (num % trillion == 0): return int_to_en(num // trillion) + ' trillion'
+    else: return int_to_en(num // trillion) + ' trillion, ' + int_to_en(num % trillion)
+
+    raise AssertionError('num is too large: %s' % str(num))
+    
+if __main__ == '__name__':
+   
+    number = int(input())
```
Comment (SKAUL05) [inline Basic-Scripts/int_to_en.py]: Ask the user for input
` number = int(input("Enter Number"))`

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

**Comment by abhip99** [comment]
Requested change has been made.

**Comment by SKAUL05** [comment]
> Requested change has been made.

@abhip99 Rebase your PR

**Review by SKAUL05 — APPROVED** [review]
Good to go @abhip99 
Review & Merge @ankitdobhal

**Pull request merged**